packages/utils/src/attach-states-to-orgs.ts

```diff
@@ -0,0 +1,70 @@
+import * as fs from "fs";
+import * as dotenv from "dotenv";
+import axios from "axios";
+
+dotenv.config();
+// Keep dotenv import and config before everything else
+
+type SimpleOrg = {
+  Id: string;
+  Name: string;
+  States?: string[];
+};
+
+type Org = {
+  Id: string;
+  Name: string;
+  AllSelected: boolean;
+  Organizations: SimpleOrg[];
+};
+
+type Orgs = {
+  AllCareQualityOrganizations: Org[];
+};
+
+const orgsList = fs.readFileSync("./orgs-list.json", "utf8");
+
+async function main() {
+  const orgs: Orgs = JSON.parse(orgsList);
+
+  let newList: SimpleOrg[] = [];
+
+  for (const org of orgs.AllCareQualityOrganizations) {
+    for (const simpleOrg of org.Organizations) {
+      try {
+        const url = `https://wpapi.sequoiaproject.org/fhir-stu3/1.0.0/Organization?_format=json&apikey=7sWkoiMqhCxR-eFJ9wQHXrmaPzfAgSTvE8YU2Gy4jbcLZNpduK&_name:contains=${simpleOrg.Name}&_radius=30&_sort=orgname&_active=true`;
```

Comment:
hell yeah - probably shouldn't hardcode the api key though

Reply:
Ah good catch i didnt see that